tests/spec/core/biblio-spec.js

```diff
@@ -212,3 +209,23 @@ describe("W3C — Bibliographic References", () => {
     expect(refs[0].textContent).toEqual("[dom]");
   });
 });
+
+it("makes sure references section has expected localization text", async () => {
+  const ops = {
+    config: makeBasicConfig(),
+    htmlAttrs: {
+      lang: "nl",
+    },
+    body: `
+    <section class="informative" id="intro">[[DOM]]</section>
+    <section>[[HTML]]</section>
+    `,
+  };
+  const doc = await makeRSDoc(ops);
+  const { textContent } = doc.querySelector("#references h2");
+  const [normRef, infoRef] = doc.querySelectorAll("#references h3");
+  expect(doc.documentElement.lang).toBe("nl");
+  expect(textContent.substr(3)).toBe("Referenties");
```

Comment:
oh, forgot about this... just test whatever the string is without doing "substr".

Reply:
would It be ok if I tested for a match like  `expect(textContent).toMatch("Referenties")` as this works actually.